orga/app/models/prescriber.js

```diff
@@ -52,6 +53,10 @@ export default class Prescriber extends Model {
     );
   }
 
+  get learnerImport() {
+    return this.features[Prescriber.featureList.LEARNER_IMPORT];
+  }
+
```

Comment:
**suggestion** : 
Ces variables sont des booleans et devraient être préfixé par `has`, `is`, `have`, `can`, ...